src/catalog/model/payment/paymentwall.php

```diff
@@ -71,4 +72,55 @@ private function prepareDeliveryData($order, $ref)
             'is_test' => $this->config->get('paymentwall_test') ? 1 : 0,
         );
     }
+
+    public function generateWidget($orderInfo, $customer, $successUrl)
+    {
+        // Init Paymentwall configs
+        $this->initConfig();
+
+        $successUrl = $this->config->get('paymentwall_success_url')

```

Comment:
let's use default success page of opencart when `paymentwall_success_url` setting is empty